tapiro/core/src/test/scala/io/buildo/tapiro/TapiroSuite.scala

```diff
@@ -0,0 +1,135 @@
+package io.buildo.tapiro
+
+import java.nio.file.Files
+
+class TapiroSuite extends munit.FunSuite {
+
+  check(
+    "http4s",
+    Server.Http4s,
+    "src/main/scala/schools/endpoints",
+    """
+      |/src/main/scala/schools/SchoolController.scala
+      |package schools
+      |
+      |case class School(id: Long, name: String)
+      |
+      |sealed trait SchoolReadError
+      |object SchoolReadError {
+      |  case object NotFound extends SchoolReadError
+      |}
+      |
+      |trait SchoolController[F[_], T] {
+      |  @query
+      |  def read(id: Long): F[Either[SchoolReadError, School]]
+      |}
+      |""".stripMargin,
+    """
+      |/src/main/scala/schools/endpoints/SchoolControllerTapirEndpoints.scala
```

Comment:
line starting with a single `/` declare files that are written to disk (in a temp dir)